rules/co-creation-patterns/rule.md

```diff
@@ -75,10 +75,17 @@ We can avoid this by using co-creation patterns. As a general rule, Pull Request
 
 ### Patterns
 
-Co-creation patterns can take several forms:
+Co-creation patterns can take some different forms:
 
-1. **Pair-programming**: Starting, reviewing and finishing a change together
-2. **Mob-programming**: Working in a small group, that collectively has all skills required
+1. **Pair-programming**: Two developers starting, reviewing and finishing a change together
+2. **Mob-programming**: Working in a small group, that collectively has all skills required. See <https://www.goretro.ai/post/mob-programming>
+
+For the patterns above, the similarities that there is a driver and a navigator.
```

Comment:
This doesn't sound right.  Can we fix up the grammar?

Reply:
done